src/coreclr/jit/compiler.h

```diff
@@ -3982,6 +3983,8 @@ class Compiler
                        CORINFO_ARG_LIST_HANDLE varList,
                        CORINFO_SIG_INFO*       varSig);
 
+    bool lvaInitSpecialSwiftParam(InitVarDscInfo* varDscInfo, CorInfoType type, CORINFO_CLASS_HANDLE typeHnd);
```

Comment:
Based on the naming of this method, I guess we'll be using this to do the `SwiftError*` type lookup too...

Reply:
Yeah, I was thinking that same logic could live here.